app/controllers/api/v1/unmatched_keywords_controller.rb

```diff
@@ -1,9 +1,9 @@
-class Api::V1::KeywordsController < ApplicationController
+class Api::V1::UnmatchedKeywordsController < ApplicationController
```

Comment:
Renaming KeywordsController to UnmatchedKeywordsController aligned the backend architecture with RESTful conventions. Rather than naming the controller after the frontend component, it was renamed to reflect the underlying table it manages, ensuring clearer intent, improved maintainability, and consistency across the application’s data and API structure.